pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -177,7 +180,12 @@ def __init__(
                 [override the model][pydantic_ai.Agent.override] for testing.
             end_strategy: Strategy for handling tool calls that are requested alongside a final result.
                 See [`EndStrategy`][pydantic_ai.agent.EndStrategy] for more information.
-            instrument: Automatically instrument with OpenTelemetry. Will use Logfire if it's configured.
+            instrument: Set to True to automatically instrument with OpenTelemetry,
```

Comment:
ps what I was thinking for ENV, was to borrow this setting. `OTEL_SDK_DISABLED` So, if you can detect that otel is available, and OTEL_SDK_DISABLED is not false, go ahead and set to true. wdyt?

I think technically `OTEL_SDK_DISABLED` still loads instrumentation (which would be same as `instrument=True`), just sets the tracer, metrics etc into no-op mode.

In any case it would be handy to hide code to instrument one way or another, though I certainly don't mind setting this would be more like other instrumentation if we don't have to explicitly write a flag value in our code.

Reply:
also I would like a way to default to otel semantics like this

```python
Agent.instrument_all(instrument=InstrumentationSettings(event_mode="logs"))
```

I totally understand some or even most backends don't support logs, so don't mind having to define another ENV. That said, we could consider borrowing an OTEL env to coerce the default of event_mode. 

e.g. `OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT` (which we should honor anyway) or `OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED` specifically about using logging?